packages/opam-monorepo/opam-monorepo.0.2.7/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Assemble and manage fully vendored Dune repositories"
+description: """
+The opam monorepo plugin provides a convenient interface to bridge the
+opam package manager with having a local copy of all the source
+code required to build a project using the dune build tool."""
+maintainer: ["anil@recoil.org"]
+authors: [
+  "Anil Madhavapeddy" "Nathan Rebours" "Lucas Pluvinage" "Jules Aguillon"
+]
+license: "ISC"
+homepage: "https://github.com/ocamllabs/opam-monorepo"
+bug-reports: "https://github.com/ocamllabs/opam-monorepo/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08.0"}
+  "conf-pkg-config" {build}
+]
+conflicts: [
+  "dune-build-info" {= "2.7.0" | = "2.7.1"}
+  "dune-configurator" {= "2.7.0" | = "2.7.1"}
+]
+dev-repo: "git+https://github.com/ocamllabs/opam-monorepo.git"
+build: [ "dune" "build" "-p" name "-j" jobs "@install" "@runtest" {with-test} ]
+flags: [ plugin ]
+pin-depends: [
+  [
+    "opam-0install.dev"
+    "git+https://github.com/ocaml-opam/opam-0install-solver"
+  ]
+]
```

Comment:
```suggestion
```
pin-depends are not allowed in opam-repository. Do you really need something from opam-0install dev? If so I can do a release

Reply:
Oh no, it's censored already, I simply forgot to remove it ! Let me fix that !

Reply:
Even if it's not strictly necessary since it's vendored, I think that an opam-0install release would still be useful :)